lncfg/db.go

```diff
@@ -111,7 +119,9 @@ type DatabaseBackends struct {
 // GetBackends returns a set of kvdb.Backends as set in the DB config.
 func (db *DB) GetBackends(ctx context.Context, chanDBPath string,
 	makeMacaroonBoltDB boltBackendCreator,
-	makeDecayedLogBoltDB boltBackendCreator) (
+	makeDecayedLogBoltDB boltBackendCreator,
```

Comment:
+1 for creating `bbolt` dbs in a generic function in `kvdb` and then just setting paths here in `lncfg`.

Reply:
Went with something slightly different, let me know what you think.